swingset/src/main/java/com/nqadmin/swingset/utils/SSSyncManager.java

```diff
@@ -85,16 +85,23 @@ public void actionPerformed(final ActionEvent ae) {
 
 			try {
 				// IF THIS IS NOT CAUSED BY THE USER ACTION (IN WHICH THE FOCUS WILL BE ON THE COMBO) THERE IS NOTHING TO DO
-				if ((rowset == null) || (rowset.getRow() < 1) || (comboBox.getSelectedIndex() == -1)
-						/* || comboBox.textField == null */
-						/* || comboBox.isBoundTextFieldNull() */
-						/* || !comboBox.hasFocus() */ // with rewrite, the editor and not the combo likely has the focus
-						) {
+				if ((rowset == null) || (rowset.getRow() < 1)) {
+//				if ((rowset == null) || (rowset.getRow() < 1) || (comboBox.getSelectedIndex() == -1)
```

Comment:
I was wondering about replacing `combBox.getSelectedIndex() == -1` with `!comboBox.hasSelection()`. But I see it's caught below.